ivadomed/testing.py

```diff
@@ -172,11 +172,10 @@ def run_inference(test_loader, model, model_params, testing_params, ofolder, cud
                                                         slice_axis=imed_utils.AXIS_DCT[testing_params['slice_axis']],
                                                         kernel_dim='2d',
                                                         bin_thr=0.9 if testing_params["binarize_prediction"] else -1)
-                    preds_npy_list.append(output_nii.get_fdata().transpose(3, 0, 1, 2))
-                    gt_lst = []
-                    for gt in metadata_idx[0]['gt_filenames']:
-                        gt_lst.append(nib.load(gt).get_fdata())
-                    gt_npy_list.append(np.array(gt_lst))
+                    # TODO: Adapt to multilabel
+                    preds_npy_list.append(output_nii.get_fdata()[:, :, :, 0])
+                    gt_npy_list.append(nib.load(fname_tmp).get_fdata())
```

Comment:
These lines fix [this](https://github.com/neuropoly/ivadomed/pull/240#discussion_r440092679).
This is however not robust to multi label task --> the entire section would actually benefit from some work (my bad.. !), which could be done in a future PR I think :-)